phpstan-baseline.neon

```diff
@@ -350,6 +350,11 @@ parameters:
 			count: 1
 			path: src/bundle/Controller/ContentViewController.php
 
+		-
+			message: "#^Offset 'relation' on string\\|null on left side of \\?\\? does not exist\\.$#"
```

Comment:
This entry can be probably resolved by adding a failsafe `$page = $request->query->get('page');` as it can return `null`.

Reply:
it's not a problem because the 'get' function can return
`bool|float|int|null|string` and then we use the notation `'page' => $page['relation'] ?? 1`, i.e. like an array and this is where the problem lies

Reply:
If this is safe, then we don't have to satisfy PHPStan at all cost. Fine with me.